ui/pages/reservation-unit/[...params].tsx

```diff
@@ -101,6 +101,7 @@ export const getServerSideProps: GetServerSideProps = async ({
       QueryTermsOfUseArgs
     >({
       query: TERMS_OF_USE,
+      fetchPolicy: "no-cache",
```

Comment:
Can't we use something else here or is the no-cache mandatory?  e.g.
```
  fetchPolicy: 'network-only', // Used for first execution
  nextFetchPolicy: 'cache-first', // Used for subsequent executions
```
The data doesn't seem like it would change that often anyway.
The server side might not be that big of a deal though so if it's necessary then sure.